doc/src/fix_reaxff_species.rst

```diff
@@ -110,10 +110,10 @@ all types from 1 to :math:`N`.  A leading asterisk means all types from
 The optional keyword *element* can be used to specify the chemical
```

Comment:
On the "keyword =" line at the top of this file can you add "... or delete or delete_rate_limit"